src/components/ButtonLanguage.module.css

```diff
@@ -8,9 +8,34 @@
   background-color: var(--light-grayish-cyan-background);
   color: var(--dark-cyan);
   font-weight: bolder;
+}
+
+.btn_close {
+  display: inline-block;
+  margin: 0.8rem 1rem 0 0;
+  border: none;
+  cursor: pointer;
+  border-radius: 0.3rem;
+  padding: 0.7rem 1rem;
```

Comment:
Why do we need to repeat the same CSS styling we already have for this component?